processing-scripts/04-obtaining-bridge-coordinates/01-obtain-bridge-split-info.py

```diff
@@ -15,127 +21,146 @@ def setup_logging():
         handlers=[logging.FileHandler("debug.log"), logging.StreamHandler()],
     )
 
-
 def load_geojson(file_path):
-    with open(file_path, "r") as f:
-        data = json.load(f)
-    return data
-
+    try:
+        with open(file_path, "r") as f:
+            data = json.load(f)
+        return data
+    except Exception as e:
+        logging.error(f"Error loading GeoJSON file: {e}")
```

Comment:
except FileNotFoundError:
        logging.error(f"GeoJSON file not found: {file_path}")
    except json.JSONDecodeError:
        logging.error(f"Error decoding JSON from file: {file_path}")
    except Exception as e:
        logging.error(f"Unexpected error loading GeoJSON file: {e}")
    return None